### DISCUSSION TITLE
XPath 2.0 - please support it

### DISCUSSION THREAD

**Discussion by traviswdev** [aclap-dev/video-downloadhelper#386] [Help and Q&A]
### Your question

Please allow us to use XPath 2.0 for smart naming rules. 

In addition, some documentation around this would be really helpful for people. 
At least link to a website that has documentation and link to a plugin that verifies xpath expressions, such as https://chromewebstore.google.com/detail/xpath-helper/hgimnogjllphhhkhlmebbmlgjoejdpjl

### Addon & CoApp version and details

I'm actually using beta, but the "copy" button doesn't work. 

https://chromewebstore.google.com/detail/xpath-helperversion: 8.2.2.9
channel: stable
build date: 2024-03-19
build options: linuxlic: false, noyt: true, browser: chrome
lang: en-US
license: accepted
key (only 16 first characters): 3751587795a9438e
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/123.0.0.0 Safari/537.36
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\Travis",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"
}

**Comment by stelliosbonadurer** [comment]
It's silly because the older version did, and you can switch to the legacy AI which still has the option but it doesn't work.

**Comment by GrampaWildWilly** [comment]
You'll want to read #838.  Be sure you click anything that says, "Show x previous replies."
